### PR TITLE
Updating the WebSdk version for release/2.1.4xx

### PR DESCRIPTION
**Pull request by vijayrkn**
Feed info - https://dotnet.myget.org/feed/dotnet-web/package/nuget/Microsoft.NET.Sdk.Web/2.1.400-preview1-20180614-1774926

@natemcmaster @mlorbetske

### CODE REVIEW

File: build/DependencyVersions.props [hunk 1]
```diff
@@ -25,7 +25,7 @@
     <MicrosoftNETSdkPackageVersion>1.0.0-preview-62924-09</MicrosoftNETSdkPackageVersion>
```
Comment (vijayrkn) [inline build/DependencyVersions.props]: @dsplaisted - Is the versioning for dotnet/sdk package different? Shouldn't this be 2.1.400-x?

**Comment by natemcmaster** [comment]
LGTM. :shipit:

File: build/DependencyVersions.props (hunk 1 above)
Comment (dsplaisted) [reply build/DependencyVersions.props]: We had a bug for a while where the package versions went back to 1.0.0.  The latest packages out of the 4xx branch should have a 2.1.400 version.

**Comment by vijayrkn** [comment]
The tests are failing with this error:
      10>/mnt/resource/j/workspace/dotnet_cli/release_2.1.4xx/debug_centos7.1_x64_prtest/build/sdks/sdks.csproj : error NU1102: 

> Unable to find package Microsoft.NET.Sdk.Web with version (>= 2.1.400-preview1-20180614-1774926)

[/mnt/resource/j/workspace/dotnet_cli/release_2.1.4xx/debug_centos7.1_x64_prtest/build/RestoreDependency.proj]
10:51:50     10>/mnt/resource/j/workspace/dotnet_cli/release_2.1.4xx/debug_centos7.1_x64_prtest/build/sdks/sdks.csproj : error NU1102: - Found 39 version(s) in BlobFeed [ Nearest version: 2.1.301 ] .

The packages are available in the myget feed - https://dotnet.myget.org/feed/dotnet-web/package/nuget/Microsoft.NET.Sdk.Web/2.1.400-preview1-20180614-1774926

Has there been a change in where cli is looking for the packages?

**Comment by dsplaisted** [comment]
Also @johnbeisner to comment on the feeds used

**Comment by livarcocc** [comment]
We haven't had this feed in a long time on the CLI, because the web sdk has been flowing through the blob feed. Any reason why we are not continuing to use the blob feed for the web sdk packages? We should be converging on the blob feed and not moving away from it.

In any case, if for now the feed is the solution, this feed need to be added to NuGetConfigFile.targets.

**Comment by vijayrkn** [comment]
We are using the blob feed when the payload is from 3xx branch through ProdCon. Is this 4xx branch already hooked into ProdCon?

**Comment by vijayrkn** [comment]
@mmitche Is there a build configuration that runs on this 2.1.4xx branch? If so, what is the way to trigger it? We want the payload from websdk (2.1.4xx) branch to flow to cli (2.1.4xx) branch through blob feed.

**Comment by vijayrkn** [comment]
@dotnet-bot
test Ubuntu x64 Release Build  please

**Comment by vijayrkn** [comment]
All tests have passed.

@livarcocc @dsplaisted  - Can you please approve so that this can be merged?

**Comment by dsplaisted** [comment]
@MattGertz FYI code flow from web sdk into CLI

**Comment by vijayrkn** [comment]
Thanks!

**Pull request merged**